static/js/app.js

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
please leave this file unformatted, I don't like unrelated style changes. Except the wss change, of course :-)